component/componentprofiles/config.go

```diff
@@ -3,13 +3,23 @@
 
 package componentprofiles // import "go.opentelemetry.io/collector/component/componentprofiles"
 
-import "go.opentelemetry.io/collector/component"
+import (
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pipeline"
+)
 
+// nolint
```

Comment:
Why does this need a nolint?

Reply:
The function below this returns a `component.DataType` which is deprecated.  The linter complains we are using it, but we need to use it.